composer/lib/dependabot/composer/update_checker/version_resolver.rb

```diff
@@ -239,9 +238,9 @@ def updated_version_requirement_string
 
           # If the original requirement is just a stability flag we append that
           # flag to the requirement
-          return "<=#{latest_allowable_version}#{lower_bound.strip}" if lower_bound.strip.start_with?("@")
+          return "==#{latest_allowable_version}#{lower_bound.strip}" if lower_bound.strip.start_with?("@")
```

Comment:
is it possible to add more/similar tests from the earlier PR to increase coverage?